concrete/src/Validation/CSRF/Token.php

```diff
@@ -71,35 +94,41 @@ public function getParameter($action = '')
     {
         $hash = $this->generate($action);
 
-        return 'ccm_token=' . $hash;
+        return static::DEFAULT_TOKEN_NAME . '=' . $hash;
     }
 
     /**
-     * Validates against a given action. Basically, we check the passed hash to see if
+     * Validate a token against a given action.
+     *
+     * Basically, we check the passed hash to see if:
      * a. the hash is valid. That means it computes in the time:action:pepper format
      * b. the time included next to the hash is within the threshold.
      *
-     * @param string $action
-     * @param string $token
+     * @param string $action The action that should be associated to the token
+     * @param string $token The token to be validated (if empty we'll retrieve it from the current request)
      *
      * @return bool
      */
     public function validate($action = '', $token = null)
     {
         if ($token == null) {
-            $token = isset($_REQUEST['ccm_token']) ? $_REQUEST['ccm_token'] : '';
+            $app = Application::getFacadeApplication();
+            $request = $app->make(Request::class);
+            $token = $request->get(static::DEFAULT_TOKEN_NAME);
```

Comment:
@mlocati I think it might be the `$request->get()`, is that just for `$_GET` or does it also do `$_POST`?